src/views/DatePickerListView.ts

```diff
@@ -9,22 +13,39 @@ export default class DatePickerListView extends DatePickerBaseView
 
   constructor(model: DatePickerFactory, continuousScroll = true) {
     super();
-
     this.continuousScroll = continuousScroll;
     this.model = model;
-    this.containerView = this.initContainerView();
-    this.populateView();
-  }
+    // initFrameView is in the super
+    this.frameElement = this.initFrameView(continuousScroll);
+    this.frameElement.className += " date-picker-list";
 
-  populateView(model: DatePickerFactory = this.model, containerView: HTMLElement = this.containerView) {
+  }
 
-    const arr = model.dateArray(12)
-    
-    arr.forEach((date: AtomicDateObject) => {
-      const el = document.createElement('div')
-      el.innerHTML = date.viewString
-      containerView.appendChild(el)
-    })
+  protected populateView(
+    model: DatePickerFactory,
+    containerView: HTMLElement,
+    frameElement: HTMLElement
+  ) {
+    throw new Error("Override populateView");
+  }
```

Comment:
Assuming this is just in progress, but why an error? Also curious why this is protected while its counterpart below in 12-hour view isn't. Mostly just confessing my confusion with TypeScript I think...

Reply:
Great question! This function must be overridden by the class that extends DatePickerListView, so this error forces anyone (future me) to actively consider what populateView should do _per_ implementation. It's a convention most commonly seen with Abstract classes.

Reply:
Thanks! I kind of figured out _what_ TypeScript abstract classes do in [pull request #5](https://github.com/kayernyc/configurable-date-picker/pull/5), but I was still unsure about _why_ you'd use them.